src/screens/HomeScreen.js

```diff
@@ -68,15 +233,98 @@ class HomeScreen extends React.Component {
           onPress={() => this.props.navigation.navigate("Badges")}
         />
         <Button
-          text="click here for modal"
+          text="click here to edit your avatar"
           onPress={() => this.toggleEditState()}
         />
+            
         <BottomSheet onDismiss={() => this.toggleEditState()} visible={editMode} height={modalHeight}>
-          <Text>Modal Child</Text>
+          <View style={styles.tabContainer}>
+            <TouchableOpacity onPress={()=>this.showImageFunc()}>
+              <Text style={{ fontSize: 25, color:this.state.hairTextColour, textDecorationLine: this.state.hair }}>Hair</Text>
+            </TouchableOpacity>
+            <TouchableOpacity onPress={()=>this.showEyeImageFunc()}>
+              <Text style={{ fontSize: 25, color:this.state.eyeTextColour, textDecorationLine: this.state.eye }}>Eyes</Text>
+            </TouchableOpacity>
+            <TouchableOpacity onPress={()=>this.showMouthImageFunc()}>
+              <Text style={{ fontSize: 25, color:this.state.mouthTextColour, textDecorationLine: this.state.mouth }}>Mouth</Text>
+            </TouchableOpacity>
+          </View>
+            
+          <View style={styles.imageTabContainer}>
+            {this.state.showImage &&
+              <View>
+                <TouchableOpacity onPress={()=>this.selectTopImage(0)}>
```

Comment:
Im just nitpicking at this point, will leave it to you whether you want to address or not

Where are these numbers coming from? What do they represent? `this.selectTopImage(0)}`

Can you either:

1. Drop a comment saying what selectTopImage(0), selectTopImage(1) mean / Or just out the data structure?
i.e. 
```
// selectTopImage(0) -> blah blah blah
// selectTopImage(1) -> blabbity blabbity blah
```
2. Or move these numbers to Constants.js and give it a meaningful name

Reply:
Definitely a good point, we do need to add comments. This section also need to be changed to use the database.

But since I am profile is dependent on this, I am merging it and open an issue for Anirudh's request.

Reply:
Asma, You merged it before merging in the latest master to here. Did you check if it would cause any merge conflicts or errors?

Reply:
Also, did you squash and merge?